### ISSUE TITLE
BFAPI causes insane memory usage.

### ISSUE THREAD

**Issue by TheGatesScratch** [BradBot1/BradsFabricApi#5]
Hi, I just installed the Fabric Sit mod, which installs BFAPI automatically (I think?). But the library literally causes the server to need an extra 2 gb! I uninstalled it because we need the resources.

**Comment by BradBot1** [comment]
Damn, i am unsure where that draw is coming from but I will give it a test, thanks for the report

**Comment by BradBot1** [comment]
Hey, I see what you mean to some extent (there is a slow increase of memory) but my profiler is not showing any major problems and the jvm garbage collector eventually clears them, maybe increase your collection rate with some jvm arguments?

**Comment by BradBot1** [comment]
Hey, have you tried using different arguments? If you want some good ones I can send you mine (they are just the optimal ones that most servers should use)

**Comment by TheGatesScratch** [comment]
> Hey, have you tried using different arguments? If you want some good ones I can send you mine (they are just the optimal ones that most servers should use)

I'm sorry for not being active :P 
No, I haven't tried different arguments yet. btw, we are running some other pretty powerful mods such as Origins and BetterEnd. but they run fine.

**Comment by TheGatesScratch** [comment]
No, sorry, I don't really have the time, and since I'm not really using this right now, it's on low priority right now...
